Mathlib/Topology/Homotopy/Path.lean

```diff
@@ -207,15 +207,8 @@ def reparam (p : Path x₀ x₁) (f : I → I) (hf : Continuous f) (hf₀ : f 0
     · rw [Set.mem_singleton_iff] at hx
       rw [hx]
       simp [hf₁]
-  continuous_toFun := by
-    -- Porting note: was `continuity` in auto-param
-    refine continuous_const.path_eval ?_
-    apply Continuous.subtype_mk
-    apply Continuous.add <;> apply Continuous.mul
-    · exact continuous_induced_dom.comp (unitInterval.continuous_symm.comp continuous_fst)
-    · continuity
-    · continuity
-    · continuity
+  -- TODO: can the auto-param be converted to use `fun_prop` (instead of `continuity`)?
```

Comment:
The current auto-param uses `continuity`, which is slower... so, a good follow-up would be to make that use `fun_prop` as well. (This causes some breakage, which is why I'd like to not do so in this PR.)